sample-status-updater-infrastructure/src/main/groovy/life/qbic/samplestatusupdater/search/SampleSearchConnector.groovy

```diff
@@ -16,9 +18,9 @@ class SampleSearchConnector implements OpenBisSearchService {
     }
 
     @Override
-    List<String> findNewOpenBisSamplesSince(Date registeredSince, List sampleTypeFilter) {
+    List<String> findNewOpenBisSamplesSince(Instant registeredSince, List sampleTypeFilter) {
         def criteria = new SampleSearchCriteria()
-        criteria.withRegistrationDate().thatIsLaterThanOrEqualTo(registeredSince)
+        criteria.withRegistrationDate().thatIsLaterThanOrEqualTo(Date.from(registeredSince))
```

Comment:
Sad, that openbis still uses `java.util.Date` but I think it is very good that we do not! 😄